packages/snaps-utils/src/test-utils/common.ts

```diff
@@ -1,8 +1,14 @@
-import { SemVerVersion } from '@metamask/utils';
+// TODO: Import from root.
+import { hexToBytes, SemVerVersion } from '@metamask/utils';
 
 export const TEST_SECRET_RECOVERY_PHRASE =
   'test test test test test test test test test test test ball';
 
+// `mnemonicPhraseToBytes(TEST_SECRET_RECOVERY_PHRASE)`.
```

Comment:
Is there a reason to define it statically or could we just call `mnemonicPhraseToBytes` instead?